README.md

```diff
@@ -37,7 +43,24 @@ The easiest way to try the web app is to run it through docker:
 - npm
 - HDF5 and NetCDF4 headers, if available on your platform
 
-On Debian, e.g., install these packages:
+
+### MacOS
+You can install these dependencies using brew. Don’t know what [brew](http://brew.sh/) is?
+
+Run:
+`
+brew install supervisor nginx postgresql npm hdf5
+`
+
+An alternative to installing these dependencies is to use pip. You can install each package by using the following:
```

Comment:
@sonajeswani suggested pip, my bad for brew I think it should 
`brew install supervisor nginx postregsql node homebrew/science` I'll try running the command before making the change.